helm/cluster-api-provider-azure/templates/v1_serviceaccount_azureserviceoperator-default.yaml

```diff
@@ -1,6 +1,9 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
+  annotations:
+    azure.workload.identity/client-id: '{{ .Values.azureWorkloadIdentity.clientId
+      }}'
```

Comment:
hmmm does it cause any issues that the annotations are broken up in two lines?

Reply:
Nope, that works perfectly fine. No worries! We also have it in other places for longer.